CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ include(GoogleTest)
 
 # Specify settings for clang-tidy
 string(REGEX REPLACE "[\\\\|/]" "[\\\\/]" HEADERS "${CMAKE_BINARY_DIR}/../.*")
-set(CMAKE_CXX_CLANG_TIDY
+set(CMAKE_CXX_CLANG_TIDY__
```

Comment:
Does this have to be disabled for the entire repo?

Reply:
I'm approving the changes but perhaps we can remove this requirement. We can disable clang-tidy for specific folders if you need.